pkg/jtracer/jtracer_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2023 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jtracer_test
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"github.com/opentracing/opentracing-go"
+)
+
+func TestOT(t *testing.T) {
+	mockTracer := opentracing.NoopTracer{}
+	jtracer := jtracer.OT(opentracing.NoopTracer{})
+
+	if jtracer.OT != mockTracer {
+		t.Errorf("Expected JTracer.OT to be equal to mockTracer")
+	}
```

Comment:
use assert from stretchr/testify

```suggestion
	assert.Equal(t, mockTracer,  jtracer.OT)
```